Cargo.lock

```diff
@@ -922,21 +925,20 @@ dependencies = [
 
 [[package]]
 name = "moka"
-version = "0.12.8"
+version = "0.12.10"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "32cf62eb4dd975d2dde76432fb1075c49e3ee2331cf36f1f8fd4b66550d32b6f"
+checksum = "a9321642ca94a4282428e6ea4af8cc2ca4eac48ac7a6a4ea8f33f76d0ce70926"
 dependencies = [
  "crossbeam-channel",
  "crossbeam-epoch",
  "crossbeam-utils",
- "once_cell",
+ "loom",
```

Comment:
Pretty sure loom is not supposed to be a lib dependency. IIRC most users guard with a `cfg` flag.

Reply:
`cargo tree | grep loom` and `cargo tree --invert loom` both produce no results. I assume this is something transitive and not our issue?

Reply:
It looks like it's hickory-resover -> moka -> loom ?

Reply:
https://github.com/moka-rs/moka/issues/501